doc/source/parameters/dem/insertion_info.rst

```diff
@@ -63,20 +63,16 @@ In this subsection, insertion methods which are ``uniform``, ``non_unifom``, ``p
 
-* The ``insertion frequency`` parameter defines the frequency of insertion. For example, if the ``insertion frequency`` is set equal to 10000, the iterations 1, 10001, 20001, ... will be defined as insertion steps.  The ``insertion frequency`` should be selected adequately depending on the insertion method. For ``uniform`` and ``non_uniform`` it should be large, so that the inserted particles at the previous insertion step have enough time to leave the insertion box for the next time step, otherwise large overlap may occur which leads to a large velocity of particles. For the ``plane`` method, it should be small so that particles are being inserted as soon as a cell is empty.
+* The ``insertion frequency`` parameter defines the frequency of insertion. For example, if the ``insertion frequency`` is set equal to 10000, the iterations 1, 10001, 20001, ... will be defined as insertion steps.  The ``insertion frequency`` should be selected adequately depending on the insertion method. For ``volume`` it should be large, so that the inserted particles at the previous insertion step have enough time to leave the insertion box for the next insertion step, otherwise large overlap may occur which leads to a large velocity of particles. For the ``plane`` method, it should be small so that particles are being inserted as soon as a cell is empty.
 
-* The ``random number range`` and ``insertion random number seed`` parameters determine the random added values to the positions of particles during a ``non_uniform`` and ``plane`` insertion. ``random number range`` defines the maximum value for the random displacement. ``insertion random number seed`` is the seed for the random number generator.
+* The ``random number range`` and ``insertion random number seed`` parameters determine the random added values to the positions of particles during a ``volume`` and ``plane`` insertion. ``random number range`` defines the maximum value for the random displacement. ``insertion random number seed`` is the seed for the random number generator.
 
-.. note::
-    The ``random number range`` and ``insertion random number seed`` are only being use by ``non_uniform`` and ``plane`` insertion methods.
-
-------------------------
-Uniform and Non-uniform
-------------------------
-
-Uniform and Non-uniform insertion methods are really similar in their functioning. Both methods use an insertion box where particles will be insert. In ``uniform`` insertion, the insertion locations of the particles inside the insertion box are tidy. In ``non_uniform`` insertion however, the insertion locations of particles are randomly selected.
+-------
+Volume
+-------
+The volume insertion methe uses an insertion box where particles will be insert. the insertion locations of particles are randomly selected if the ``random number range`` is not equal to zero, otherwise, the particles will be perfecly aligns with respect to the x, y and z directions.
```

Comment:
```suggestion
The volume insertion method uses an insertion box where particles will be inserted. the locations of particles are randomly selected if the ``random number range`` is not equal to zero, otherwise, the particles will perfectly align with the x, y and z directions.
```